packages/java/src/browser/java-client-contribution.ts

```diff
@@ -111,9 +107,25 @@ export class JavaClientContribution extends BaseLanguageClientContribution {
         });
     }
 
-    protected createOptions(): LanguageClientOptions {
+    async activate(): Promise<Disposable> {
```

Comment:
please make sure to call `super.activate` to make sure that restarting works properly, it was changed recently